### ISSUE TITLE
When a new user joins, they get three welcome messages

### ISSUE THREAD

**Issue by lpatmo** [codebuddiesdotorg/greetbot#63]
When a new user signs up, they'll see the "welcome" message fire thrice.

Some people want to report this as a bug.

**Comment by lpatmo** [comment]
I've created a new user and can verify that this happens.

**Comment by stain88** [comment]
Is this a new user to codebuddies, or the greetbot slack?
If codebuddies, I can't reproduce. If greetbot, it's probably because we've "duped" the bots for individual testing, so all of them fire.

**Comment by BethanyG** [comment]
Confirm that a new user to **_codebuddies.slack.com_** gets (_in my case_) two bot instances and two Welcome messages.  This was after I invited the user via an **email/link** from the Slack interface.  **However** @stain88 used the "self-invite" method (the [get your invite] button on the website) - and only received a single message on joining. 

I removed the second bot configuration for codebuddies.  With one bot, I am only getting one welcome message (unless I click the "how does this app work?" link in the welcome message -- then I get two.

Currently, this deactivates the `/welcome` functionality for the bot (the current configuration doesn't have `/` commands configured.  However, this will be fixed when we deploy the completed `/resources` functionality.

Can someone re-test, and reconfirm?  Thanks!
